compiler/rustc_mir_build/src/build/coverageinfo.rs

```diff
@@ -118,17 +118,37 @@ impl BranchInfoBuilder {
         }
     }
 
-    fn add_two_way_branch<'tcx>(
+    fn register_two_way_branch<'tcx>(
         &mut self,
+        tcx: TyCtxt<'tcx>,
         cfg: &mut CFG<'tcx>,
         source_info: SourceInfo,
         true_block: BasicBlock,
         false_block: BasicBlock,
     ) {
-        let true_marker = self.markers.inject_block_marker(cfg, source_info, true_block);
-        let false_marker = self.markers.inject_block_marker(cfg, source_info, false_block);
+        let this = self;
```

Comment:
Nitpick: Any particular reason for using `this` instead of just `self`?

Reply:
I wanted to stay consistent with the rest of the `rust_mir_build` code, which justifies the use of `this` like [this](https://github.com/rust-lang/rust/blob/a330e49593ee890f9197727a3a558b6e6b37f843/compiler/rustc_mir_build/src/build/expr/into.rs#L27-L29).

> since we frequently have to reference `self` from within a
> closure, where `self` would be shadowed, it's easier to
> just use the name `this` uniformly.

In this specific case, we're only using this once in a closure though, so we might as well get rid of it.
What do you think ?

Reply:
Ah, I believe that convention is specifically for the case where `this` is one of the closure’s *parameters*, so it can’t be named `self`. In that scenario, consistently using `this` outside and inside is helpful.

That doesn’t apply here, so I think we can just get rid of it.